### PR TITLE
Separate load data retrieval from processing with new rule `retrieve_load_data`

### PR DESCRIPTION
**Pull request by fneum**

### CODE REVIEW

File: Snakefile [hunk 1]
```diff
@@ -67,7 +67,14 @@ if config['enable'].get('retrieve_databundle', True):
         script: 'scripts/retrieve_databundle.py'
 
 
+rule retrieve_load_data:
+    input: HTTP.remote("data.open-power-system-data.org/time_series/2019-06-05/time_series_60min_singleindex.csv", keep_local=True, static=True)
```
Comment (FabianHofmann) [inline Snakefile:71]:
```suggestion
    input: HTTP.remote(config["load"]["url"], keep_local=True, static=True)
```
Shouldn't we leave the url adress in the config and not hard code it here?

**Review by FabianHofmann — COMMENTED** [review]
Looks sensible, just one remark

File: Snakefile (hunk 1 above)
Comment (euronion) [reply Snakefile:71]: We discussed it locally and found it to be more comprehensible, if the input data (i.e. also the URL) is located in the Snakefile, as all the other data which we retrieve from the internetz.

Comment (FabianHofmann) [reply Snakefile:71]: I agree, all data paths (even remote ones) are in the Snakefile, model setting in the config. Sounds good

**Pull request merged**